pkgs/os-specific/linux/kernel/manual-config.nix

```diff
@@ -188,7 +188,8 @@ let
       postInstall = (optionalString installsFirmware ''
         mkdir -p $out/lib/firmware
       '') + (if (platform ? kernelDTB && platform.kernelDTB) then ''
-        make $makeFlags "''${makeFlagsArray[@]}" dtbs dtbs_install INSTALL_DTBS_PATH=$out/dtbs
+        make -j$NIX_BUILD_CORES $makeFlags "''${makeFlagsArray[@]}" dtbs INSTALL_DTBS_PATH=$out/dtbs
+        make -j$NIX_BUILD_CORES $makeFlags "''${makeFlagsArray[@]}" dtbs_install INSTALL_DTBS_PATH=$out/dtbs
```

Comment:
What's the reason for splitting the make commands?

Reply:
Since the dependencies aren't specified well, it will otherwise try to install dtbs before having built them, causing the build to fail.

Reply:
You should probably add a comment about that

Reply:
is this known upstream ?

Reply:
I suspect it's a WONTFIX, installing the image or modules with the same make invocation as building them will fail in the same way AFAIK.

Reply:
as @atemu it is confusing enough to warrant a comment

Reply:
I'm reluctant to comment this here, because there's a huge amount of software that has this issue and documenting it only for linux would be weird. What do you think of the alternative approach I've pushed here?